frame/democracy/src/lib.rs

```diff
@@ -277,6 +279,15 @@ pub trait Trait: frame_system::Trait + Sized {
 
 	/// The Scheduler.
 	type Scheduler: ScheduleNamed<Self::BlockNumber, Self::Proposal>;
+
+	/// The maximum number of possible vetoers origin, not a hard limit, only used to compute weight.
+	type MaxVetoers: Get<u32>;
```

Comment:
This counts the number of people in the `VetoOrigin`? Can't we get this data from the `VetoOrigin` trait?

Reply:
Indeed could gather information into VetoOrigin, probably an additional trait, https://github.com/paritytech/substrate/pull/5802 should also make it easier to put a maximum on the number of member of collective